les/clientpool_test.go

```diff
@@ -102,9 +104,11 @@ func testClientPool(t *testing.T, connLimit, clientCount, paidCount int, randomD
 
 		if tickCounter == testClientPoolTicks/4 {
 			// give a positive balance to some of the peers
-			amount := uint64(testClientPoolTicks / 2 * 1000000000) // enough for half of the simulation period
+			amount := int64(testClientPoolTicks / 2 * 1000000000) // enough for half of the simulation period
```

Comment:
I think numbers with so many zeroes are bad in code - wonder if go can do something like 1_000_000_000 or alternative 10**9

Reply:
its real meaning is nanoseconds per second so I figured I can use time.Second here